src/components/views/AgreementForm.js

```diff
@@ -147,10 +147,10 @@ class AgreementForm extends React.Component {
     const endButton = (
       <Button
         buttonStyle="primary mega"
-        disabled={this.props.pristine || this.props.submitting || this.props.invalid}
+        disabled={pristine || submitting}
```

Comment:
May want to add _invalid_ as well? Then when you render an empty card on clicking the _Add license to agreement_ button, the _save and close_ button would still be disabled.

Reply:
So I actually remove `invalid` on purpose...

If you link a license, then you have to set that license's status and the status of its amendments. However, `Select` doesn't show the `error` it gets until the field is `touched`. So if you just add a license, then the Save button is disabled and you have no idea why unless you notice the little required asterisk.

Now that I've enabled the button if the form is invalid, when the user clicks Save, the errors will get shown and the form will scroll to the errors themselves. This is much more friendly to the user. I think we should consider making this change in ui-licenses and ui-local-kb-admin as well...

That's not really strong UX IMO, so

Reply:
Ah okay. That makes sense. Cool.